### ISSUE TITLE
Mnemonic is empty

### ISSUE THREAD

**Issue by karan** [nextcloud/desktop#2285]
### Expected behaviour

I should see the E2E mnemonic.

### Actual behaviour

It's empty: 

![image](https://user-images.githubusercontent.com/3261985/90592526-a2288880-e19a-11ea-84f5-97dfb0605771.png)

### Steps to reproduce
1. Install Mac client 3.0
2. Open the client, and note that it says "This account supports end-to-end encryption"
3. Click the "Display mnemonic" button

### Client configuration
Client version: 3.0.0

Operating system: MacOS 10.15.5

OS language: English

Qt version used by client package (Linux only, see also Settings dialog):

Client package (From Nextcloud or distro) (Linux only):

Installation path of client:

### Server configuration

Nextcloud version: 18.0.7

Storage backend (external storage): Local disk

### Logs

Please use Gist (https://gist.github.com/) or a similar code paster for longer
logs.

1. Client logfile: Output of `nextcloud --logwindow` or `nextcloud --logfile log.txt`
(On Windows using `cmd.exe`, you might need to first `cd` into the Nextcloud directory)
(See also https://docs.nextcloud.com/desktop/2.3/troubleshooting.html#log-files)

2. Web server error log:

3. Server logfile: nextcloud log (data/nextcloud.log):

**Comment by er-vin** [comment]
I guess your server doesn't have the end_to_end_encryption app installed?

If it is indeed not installed, I will close this one it has been fixed by #2280 and #2281. That fix will be out in 3.0.1 which should be tagged end of next week if everything goes well. We had sometimes false positives in that part of the GUI wrongly assuming e2ee was available when it wasn't.

**Comment by devcircus** [comment]
Seeing the same thing on my windows client. E2E is enabled and the apps installed on the server. Look forward to the fix release.

**Comment by er-vin** [comment]
> Seeing the same thing on my windows client. E2E is enabled and the apps installed on the server. Look forward to the fix release.

It's something different if you got E2EE app installed and enabled and in version 1.5.2 or above. If that's the case please open a separate issue with as many details as possible (including debug logs).

**Comment by karan** [comment]
I just enabled the e2ee app 1.4.0 on my server, restarted the client and I still have the same behavior

<img width="713" alt="Screen Shot 2020-08-19 at 12 32 15 PM" src="https://user-images.githubusercontent.com/3261985/90681309-0d647000-e218-11ea-8de3-c5c70727547c.png">

Are there any settings for e2ee app I need to configure? I couldn't find any documentation, and can't find the option on the server either.

I see 1.6.1 was released yesterday for the e2ee app but it's not yet available in the app store? What's the normal latency?
